cargo-dylint/src/main.rs

```diff
@@ -116,6 +116,12 @@ struct Dylint {
 
     #[clap(last = true, help = "Arguments for `cargo check`")]
     args: Vec<String>,
+
+    #[clap(long, value_name = "path", help = "Path to file to pipe stderr to.")]
```

Comment:
```suggestion
    #[clap(long, value_name = "path", help = "Path to file to pipe stderr to")]
```
To be consistent with the other option descriptions.